sources/procfs.go

```diff
@@ -675,6 +675,31 @@ func parseFileElements(path string, directoryDepth int) (name string, nodeName s
 	return name, nodeName, nil
 }
 
+func convertToBytes(s string) string {
+	new_s := ""
+	multiplier := 1
+	switch finalChar := s[len(s)-1:]; finalChar {
```

Comment:
Again, there _shouldn't_ be an issue with this, but might be safe to force this to an uppercase value. Might add a bit of safety just in case these values get changed for some odd reason in future updates. Though I certainly don't feel very strongly about this, so take it or leave it as you desire.